roles/installer/tasks/main.yml

```diff
@@ -63,10 +63,7 @@
     pod: "{{ tower_pod_name }}"
     container: "{{ meta.name }}-task"
     command: >-
-      bash -c "echo 'from django.db import connection;
-      tbl = \"auth_user\" in connection.introspection.table_names();
-      exit(0 if tbl else 1)'
-      | awx-manage shell"
+      bash -c "awx-manage showmigrations | grep -v '[X]' | grep '[ ]' | wc -l"
```

Comment:
Without this, we were only running migrations when the auth_user table didn't exist.  This would not have worked for upgrades, the resulting Tower would not apply any migrations added.